build-system/babel-plugins/babel-plugin-transform-jss/test/fixtures/transform-assertions/ignore-unrelated-exports/output.mjs

```diff
@@ -0,0 +1,25 @@
+/** @enum {string}*/
+var _classes = {
+  floatLeft: "float-left-07984bd"
+};
+
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export const useStyles = () => _classes; // These next lines should be unaffected by jss transform.
```

Comment:
comment gets placed in the wrong location -- which should be harmless

Reply:
Yah, this is an annoying Babel thing. It attaches as a trailing comment to the _last_ node, where I would want it to attach as a leading comment to the next node.